specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2020-11-01-preview/appplatform.json

```diff
@@ -0,0 +1,4634 @@
+          "type": "string"
+        },
+        "uri": {
+          "description": "The uri of the config server settings error record",
+          "type": "string"
+        },
+        "messages": {
+          "description": "The detail error messages of the record",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ConfigServerResource": {
+      "description": "Config Server resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ConfigServerProperties",
+          "description": "Properties of the Config Server resource",
+          "x-ms-client-flatten": false
+        }
+      }
+    },
+    "ConfigServerProperties": {
+      "description": "Config server git properties payload",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "State of the config server.",
+          "enum": [
+            "NotAvailable",
+            "Deleted",
+            "Failed",
+            "Succeeded",
+            "Updating"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ConfigServerState",
+            "modelAsString": true
+          }
+        },
+        "error": {
+          "$ref": "#/definitions/Error",
+          "description": "Error when apply config server settings."
+        },
+        "configServer": {
+          "$ref": "#/definitions/ConfigServerSettings",
+          "description": "Settings of config server."
+        }
+      }
+    },
+    "MonitoringSettingResource": {
+      "description": "Monitoring Setting resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/MonitoringSettingProperties",
+          "description": "Properties of the Monitoring Setting resource",
+          "x-ms-client-flatten": false
+        }
+      }
+    },
+    "MonitoringSettingProperties": {
+      "description": "Monitoring Setting properties payload",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "State of the Monitoring Setting.",
+          "enum": [
+            "NotAvailable",
+            "Failed",
+            "Succeeded",
+            "Updating"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "MonitoringSettingState",
+            "modelAsString": true
+          }
+        },
+        "error": {
+          "$ref": "#/definitions/Error",
+          "description": "Error when apply Monitoring Setting changes."
+        },
+        "traceEnabled": {
```

Comment:
As this is a new preview API version, why not remove this property entirely? 
To educate customers to use 'appInsightsInstrumentationKey' instead of 'traceEnabled', specify in the description of 'appInsightsInstrumentationKey'.

Reply:
We go through the breaking change review meeting and get some feed back from Jeffery Ricther, who suggest us to keep the existed property from last stable version, with some description to customer about deprecated.

Reply:
@Incarnation-p-lee Thanks for the update.

Reply:
@RamyasreeChakka Thanks a lot for reviewing.